pkg/controller/inferenceservice/resources/knative/service.go

```diff
@@ -321,3 +283,29 @@ func (c *ServiceBuilder) CreateExplainerService(name string, metadata metav1.Obj
 
 	return service, nil
 }
+
+func (c *ServiceBuilder) getCommonAnnotations(metadata metav1.ObjectMeta, minReplicas int, maxReplicas int) (map[string]string, error) {
```

Comment:
nit: I'd probably call this buildAnnotations. We're not only getting them, but taking an object, applying a blacklist, and then decorating with extras.

Reply:
Sure, updated.